### ISSUE TITLE
cannot install alongside kodi-pvr-hts

### ISSUE THREAD

**Issue by ehough** [kodi-pvr/pvr.wmc#60]
I'm trying to install this add-on in Linux, but there's a (seemingly bogus) conflict with `kodi-pvr-hts`. Here's how to reproduce:

    $ docker run -it --rm ubuntu:bionic
    # apt-get update && apt-get -y install software-properties-common
      ...
    # add-apt-repository ppa:team-xbmc/ppa && apt-get update
      ...
    # apt-get install kodi-pvr-wmc kodi-pvr-hts
      ...
     Some packages could not be installed. This may mean that you have
     requested an impossible situation or if you are using the unstable
     distribution that some required packages have not yet been created
     or been moved out of Incoming.
     The following information may help to resolve the situation:

     The following packages have unmet dependencies:
      kodi-pvr-wmc : Conflicts: kodi-pvr-tvheadend-hts
     E: Unable to correct problems, you have held broken packages.

Upon inspecting the `kodi-pvr-wmc` package, we see that it conflicts with and replaces `kodi-pvr-tvheadend-hts`:

```
# apt-cache show kodi-pvr-wmc
Package: kodi-pvr-wmc
Priority: extra
Section: libs
Installed-Size: 675
Maintainer: Nobody <nobody@kodi.tv>
Architecture: amd64
Version: 1.4.9-1~bionic
Replaces: kodi-pvr-tvheadend-hts
Depends: libc6 (>= 2.17), libgcc1 (>= 1:3.3.1), libstdc++6 (>= 5.2)
Conflicts: kodi-pvr-tvheadend-hts
Filename: pool/main/k/kodi-pvr-wmc/kodi-pvr-wmc_1.4.9-1~bionic_amd64.deb
Size: 176856
MD5sum: 6be9e14cc06c607df84e45dba8bbf070
SHA1: aca0e65dbcc0901b0f293ae87d361448695c8472
SHA256: cd2f811f3a6a58e5e4a245469de4726b0a436f1ab586d3fd1eecb9437d229f19
Description: WMC PVR for Kodi
Description-md5: 81a7d4d1a439c5977df11a5913ac0622
```

But according to [this commit](https://github.com/kodi-pvr/pvr.wmc/commit/0c591368ed96333e2dab594f18bd7530056f8512), the `Conflicts` and `Replaces` lines shouldn't be there.

Does that all make sense? It seems like the PPA has outdated package control information, but I'm not an expert in Debian packaging so I could be misguided.

**Comment by Jalle19** [comment]
The Conflicts probably shouldn't me there, can you make a pull request? Ping @wsnipex

**Comment by ehough** [comment]
Thanks for the ack.

> can you make a pull request?

I would be glad to help, but I'm really not sure where to send a PR. The code [looks right](https://github.com/kodi-pvr/pvr.wmc/blob/b62f80eda1d68c691aed3de31e833ab1f9282433/debian/control#L9), so to me it seems like a packaging bug. Any hints on where I could find the packaging/distribution code for the PPA?

**Comment by wsnipex** [comment]
https://github.com/kodi-pvr/pvr.wmc/blob/Krypton/debian/control#L13-L14
those lines need to be removed

**Comment by Jalle19** [comment]
> https://github.com/kodi-pvr/pvr.wmc/blob/Krypton/debian/control#L13-L14
> those lines need to be removed

Seems like the fix is in master, so only Krypton packaging will need patching.

**Comment by Rechi** [comment]
fixed at #46 and #61
